### PR TITLE
Completed assignment-2

### PR DESCRIPTION
**Pull request by mchoudhury6**
## What changes are you trying to make? (e.g. Adding or removing code, refactoring existing code, adding reports)
Plotted scatter plots with regression lines to visualize relationships, trained a model using numeric predictors, and calculated RMSPE.
## What did you learn from the changes you have made?
How to use RMSPE
## Was there another approach you were thinking about making? If so, what approach(es) were you thinking of?
No
## Were there any challenges? If so, what issue(s) did you face? How did you overcome it?
No
## How were these changes tested?
Running code
## A reference to a related issue in your repository (if applicable)
N/A 
## Checklist
- [Y] I can confirm that my changes are working as intended

### CODE REVIEW

**Review by KasraVakiloroayaei — APPROVED** [review]
A2: Complete 

Great work!
Q4: I recommend that you try using another function in order to find the correct RMSE value. In your code you used the following function:
"rmspe = np.sqrt(**np.mean**(((y_test - y_pred) / y_test) ** 2))"

Instead I would suggest using "**mean_squared_error**" function to calculate the RMSE. 

The arguments in RMSE would contain a vector of your **test set response** and a vector of your **y predicted value** (found on the line of best fit in the case of linear regression)

**Comment by KasraVakiloroayaei** [comment]
Just to be clear RMSE and RMSPE are sometimes used interchangeably. 

But RMSE is actual referring to amount of error in the training data after you finished modeling 
and RMSPE is referring to the amount of error in your testing data (thus the predicted) after you finish modeling the data
